CharacterCreation/01_Races.txt

```diff
@@ -32,26 +32,26 @@ more temperate, jungle climate. 90% of the annual rainfall on the planet
 occurs on this jungly southern continent.

```

Comment:
Line 28: "Upright" axis does not mesh with further data that a pole points toward the suns. Upright sounds like an axis perpendicular to the plane of the ecliptic. Also I'm not sure if a planet with that kind of tidal locking and axial direction is astronomically plausible.
Line 29: "it's" -> "its".
Line 29: Capitalize "The north pole points..."